### PR TITLE
Three tries for failing func tests

### PR DESCRIPTION
**Pull request by kzangeli**
Implements issue #1619

### CODE REVIEW

File: test/functionalTest/testHarness.sh [hunk 1]
```diff
@@ -640,6 +666,7 @@ function runTest()
   fileCreation $path $filename
   if [ "$toBeStopped" == "yes" ]
   then
+    runTestStatus="stopped2"
```
Comment (fgalan) [inline test/functionalTest/testHarness.sh]: What "stopped2" means?

File: test/functionalTest/testHarness.sh [hunk 2]
```diff
@@ -731,22 +764,54 @@ do
...
+      else
+        printf "Running test %03d/%d: %s\n" "$testNo" "$noOfTests" "$testFile"
+      fi
+
+      runTest $testFile $tryNo
+      if [ $shellResult == "0" ]
+      then
+        if [ $tryNo != 1 ]
+        then
+          echo "OK"
```
Comment (fgalan) [inline test/functionalTest/testHarness.sh]: Is the trial number printed in the case of 2nd and 3rd trial as part as the output line corresponding to the test? Maybe it can be interred from the logic script, but it is a bit hard to follow to me...

Comment (kzangeli) [inline test/functionalTest/testHarness.sh]: Yes - as you say.
The first 'informative line' is as before while the second and consecutive kines get an extra "(intent X).

Look up line 781:

    if [ $tryNo != 1 ]
      then
      tryNoInfo="(intent $tryNo)"

File: test/functionalTest/testHarness.sh (hunk 1 above)
Comment (kzangeli) [inline test/functionalTest/testHarness.sh]: "stopped2" and not "stopped", as "stopped" has already been used.
I don't even use these values. Just that it is != "OK"

File: test/functionalTest/testHarness.sh (hunk 2 above)
Comment (fgalan) [inline test/functionalTest/testHarness.sh]: NTC

File: test/functionalTest/testHarness.sh (hunk 1 above)
Comment (fgalan) [inline test/functionalTest/testHarness.sh]: NTC

**Comment by fgalan** [comment]
LGTM
